config/manifests/bases/yugabyte-platform-operator.clusterserviceversion.yaml

```
@@ -0,0 +1,63 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: Database
+    certified: "true"
+    containerImage: registry.connect.redhat.com/yugabytedb/yugabyte-platform-operator:latest
+    description: Operator for Yugabyte Platform makes it easy to deploy and upgrade Yugabyte platform on Openshift environments.
+    operators.operatorframework.io/builder: operator-sdk-v1.0.0
+    operators.operatorframework.io/project_layout: helm.sdk.operatorframework.io/v1
+    repository: https://github.com/yugabyte/yugabyte-platform-operator
+    support: YugabyteDB
+  name: yugabyte-platform-operator.vX.Y.Z
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: |2
+
+    Operator for Yugabyte Platform makes it easy to deploy and upgrade Yugabyte platform on Openshift environments.
+    Use Yugabyte Platform's orchestration and monitoring to manage YugabyteDB universes. Yugabyte Platform can create a YugabyteDB universe with many instances (VMs, pods, machines, etc., provided by IaaS), logically grouped together to form one logical distributed database. Each universe includes one or more clusters. A universe is comprised of one primary cluster and, optionally, one or more read replica clusters. All instances belonging to a cluster run on the same type of cloud provider instance type.
+
+    ### Yugabyte Platform Features
+    * **YugabyteDB-as-a-Service on Kubernetes** - Deploy your own YugabyteDB-as-a-Service running on any Kubernetes infrastructure, Yugabyte Platform provides central management for multiple database clusters and Includes 24/7 production support. * **Monitoring & Alerting** - Comprehensive monitoring & alerting without any configuration. * **Scheduled Backups** - Use Yugabyte Platform to perform a regular scheduled backup of a full YugabyteDB universe or selected tables.
```

```suggestion
    * **YugabyteDB-as-a-Service on Kubernetes** - Deploy your own YugabyteDB-as-a-Service running on any Kubernetes infrastructure, Yugabyte Platform provides central management for multiple database clusters and Includes 24/7 production support. 
    * **Monitoring & Alerting** - Comprehensive monitoring & alerting without any configuration. 
    * **Scheduled Backups** - Use Yugabyte Platform to perform a regular scheduled backup of a full YugabyteDB universe or selected tables.
```
Fixing the formatting.